.github/main.workflow

```diff
@@ -0,0 +1,16 @@
+workflow "Publish Helm chart" {
+  on = "push"
+  resolves = ["helm-gh-pages"]
+}
+
+action "helm-lint" {
+  uses = "stefanprodan/gh-actions/helm@master"
+  args = ["lint chart/flux"]
+}
+
+action "helm-gh-pages" {
+  needs = ["helm-lint"]
+  uses = "stefanprodan/gh-actions/helm-gh-pages@master"
+  args = ["chart/flux","https://weaveworks.github.io/flux","chart-"]
```

Comment:
What's the last argument -- a prefix for the tag to look for?

Reply:
Yes, as per our chart release procedure we tag the master branch with `chart-<VERSION>`. This action will only publish the chart to gh-pages when that prefix is used.

Reply:
OK, so combining with my other question: is it not possible in GitHub Actions to only run this bit if it's tagged, rather than relying on the script to know how to do that?

Reply:
Yes that's possible. I've added a tag filter before the the publish action.

Reply:
Neat!